clients/broadcaster.go

```diff
@@ -61,8 +63,13 @@ type EncodedProfile struct {
 	ChromaFormat int64  `json:"chromaFormat,omitempty"`
 }
 
-var client = &http.Client{
-	Timeout: TRANSCODE_TIMEOUT,
+var client = retryablehttp.Client{
```

Comment:
Does it make sense to use `newRetryableClient()` here as well then this way it's one function instead of having a similar initialization in two places?